src/test/java/com/capitalone/dashboard/collector/SCMHttpRestClientTest.java

```diff
@@ -88,23 +88,24 @@ public void makeRestCallAndtestBasicAuthenticationForGlobalCredentials() throws
                 classArgumentCaptor.capture()))
                 .willReturn(responseEntity);
 
-        given(settings.getKey()).willReturn("WI9XFb8cW9bxeS/fwdka0LBXFbqtwvEa");
-        given(settings.getPassword()).willReturn("password");
-        given(settings.getUsername()).willReturn("user");
+//        given(settings.getKey()).willReturn("WI9XFb8cW9bxeS/fwdka0LBXFbqtwvEa");
```

Comment:
please remove the commented lines of code

Reply:
done